common/browsered/field-validation-checks.js

```diff
@@ -14,31 +14,32 @@ const validationErrors = {
   validEmail: 'Please use a valid email address',
   https: 'URL must begin with https://',
   belowMaxAmount: `Choose an amount under £${MAX_AMOUNT.toLocaleString()}`,
-  sortCode: `Sort codes must contain 6 digits`,
-  accountNumber: `Account numbers must contain 6-8 digits`
+  sortCode: 'Sort codes must contain 6 digits',
+  accountNumber: 'Account numbers must contain 6-8 digits',
+  requiresAuthorisation: 'Please authorize'
 }
 
-module.exports.isEmpty = function (value) {
+module.exports.isNotEmpty = function (value) {
```

Comment:
isNotEmpty is upside down? It's checking if it's empty

Reply:
Yes, my understanding is that when the value is what the methods says it is, then we return `null` (no error).
In the case of `isEmpty('')` will return `'This is field cannot be blank'` (but we are asking "is `''` string empty", that is why I renamed it).

Reply:
We are going to refactor all the field validation checks to return `true` or `false` and this way it will be more clear (as mentioned in https://github.com/alphagov/pay-direct-debit-frontend/pull/40#discussion_r165375003)